test/y2add_on/clients/add-on_auto_test.rb

```diff
@@ -3,20 +3,237 @@
 require_relative "../../test_helper"
 require "add-on/clients/add-on_auto"
 
+require "byebug"
```

Comment:
ugh, this should not be in final code as we really do not want to build depend on byebug,

Reply:
I forgot to remove it :sweat_smile:  My fault. Thanks for catching it :)